tests/e2e/performance/test_single_pvc_clone_perf.py

```diff
@@ -22,35 +23,36 @@
     argnames=["interface_type", "pvc_size", "file_size"],
     argvalues=[
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '1', '600M'], marks=pytest.mark.polarion_id('OCS-2356')
+            *[constants.CEPHBLOCKPOOL, '1', '600'], marks=pytest.mark.polarion_id('OCS-2356')
```

Comment:
I'm not sure why you change 600M to 600, but 600M is easy to read :)

Reply:
@keesturam The string "600M" is used by fillio_fs method  for writing 600M to a pod. 
However, I do need int "600" for speed calculation. 

So I either define 600 as parameter and then add "M" to it while building the string for fillio_fs 
Or pass 600M but later will have to omit the "M" for speed calculation. 

The first option seemed more convenient for me, but the efforts are actually the same. 

So what do you say? Can we leave it like this or should I change it?